cmd/root.go

```diff
@@ -34,12 +42,25 @@ HPC deployments on the Google Cloud Platform.`,
 				log.Fatalf("cmd.Help function failed: %s", err)
 			}
 		},
-		Version: "v1.4.1",
+		Version:     "v1.4.1",
+		Annotations: annotation,
 	}
 )
 
 // Execute the root command
 func Execute() error {
+	if len(GitBranch) > 0 {
+		if len(GitTagVersion) == 0 {
```

Comment:
I would do something similar for git branch. If git branch is empty then you can say something like 'detached HEAD`.